Xamarin.Forms.Core/Internals/ShellExtensions.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Xamarin.Forms.Internals
+{
+	internal static class ShellExtensions
+	{
+		public static IEnumerable<BaseShellItem> GetShellDescendants(this BindableObject element)
```

Comment:
see comments about inherited BPs below. I think we can drop this all

Reply:
Dropped.